gravity.sh

```diff
@@ -359,9 +359,10 @@ gravity_DownloadBlocklists() {
   for ((i = 0; i < "${#sources[@]}"; i++)); do
     url="${sources[$i]}"
     domain="${sourceDomains[$i]}"
+    id="${sourceIDs[$i]}"
 
     # Save the file as list.#.domain
-    saveLocation="${piholeDir}/list.${i}.${domain}.${domainsExtension}"
+    saveLocation="${piholeDir}/list.${id}.${domain}.${domainsExtension}"
```

Comment:
I think the branch changes may have confused `git` a little bit too much.

Reply:
Yeah, I noticed. 
I think I need a proper git introduction... first time rebasing. Will close this one and make a new PR based on release/v5.1.2

Reply:
No worries, it's `git` so expect the unexpected.